index.html

```diff
@@ -239,11 +241,13 @@ <h6 class="title is-5">
 <script src="assets/js/lib/filters/mealsMapFilters.js"></script>
 <script src="assets/js/lib/filters/triageMapFilters.js"></script>
 
+<script src="assets/js/lib/constants.js"></script>
```

Comment:
Move the constants file above all these script files so that we can use it for more constants in the future.

Reply:
done.